### PR TITLE
Loading data docs + fix minor docs stuff

### PR DESCRIPTION
**Pull request by piotmag769**
## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/appendix/forge-library/parse_txt.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - integers in range of `[0, P)` where P is [`Cairo Prime`](https://book.cairo-lang.org/ch02-02-data-types.html?highlight=prime#felt-type)
+  - strings of length `<=31` enclosed in single or double quotation marks
+
+For example, this file content:
+```txt
+1  2   'hello' 10     "world"
+```
+will be parsed to the following array:
+```rust
+array![1, 2, 'hello', 10, 'world]
```
Comment (war-in) [inline docs/src/appendix/forge-library/parse_txt.md:33]:
```suggestion
array![1, 2, 'hello', 10, 'world']
```

File: docs/src/appendix/forge-library/deserialize_txt.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    let my_struct = TxtParser::<MyStruct>::deserialize_txt(@file).unwrap();
+    // ...
+}
+```
+
+File content must consists of elements that:
+- have to be separated with whitespaces
+- have to be either:
+  - integers in range of `[0, P)` where P is [`Cairo Prime`](https://book.cairo-lang.org/ch02-02-data-types.html?highlight=prime#felt-type)
+  - strings of length `<=31` enclosed in single or double quotation marks
```
Comment (war-in) [inline docs/src/appendix/forge-library/deserialize_txt.md:36]: why do we allow for both single and double quotation marks? Shouldn't we wait with double until strings are supported?

**Review by war-in — APPROVED** [review]

File: docs/src/appendix/forge-library/deserialize_txt.md (hunk 2 above)
Comment (piotmag769) [reply docs/src/appendix/forge-library/deserialize_txt.md:36]: I did that only to be in line with future json support, as json standard does not allow single quoted strings. Not sure if it is a good idea though cc @MaksymilianDemitraszek

Comment (war-in) [reply docs/src/appendix/forge-library/deserialize_txt.md:36]: I think we could add double quotes in the json PR

Comment (MaksymilianDemitraszek) [reply docs/src/appendix/forge-library/deserialize_txt.md:36]: I think we should use single quutation for short strings, double quoutations should be used for long strings once those are released.

**Pull request merged**